src/object/cli_obj.c

```diff
@@ -2049,6 +2071,34 @@ shard_rw_prep(struct shard_auxi_args *shard_auxi, struct dc_object *obj,
 	return 0;
 }
 
+//static void
+//obj_update_csums(const struct dc_object *obj, const daos_obj_update_t *args) {
+//	struct daos_csummer *csummer = dc_cont_hdl2csummer(obj->cob_coh);
+//
+//	if (!daos_csummer_initialized(csummer)) /** Not configured */
+//		return;
+//
+//	int i;
+//
+//	for (i = 0; i < args->nr; i++) {
+//		/**
+//		 * TODO: Turn this into an assert after csums are
+//		 * removed from public interface. csums should always be
+//		 * cleaned up/freed internally, so might be a memory leak if
+//		 * not NULL. But user can pass in right now so don't assert.
+//		 * (D_ASSERT(args->iods[i].iod_csums == NULL)
+//		 */
```

Comment:
(style)  Block comments use a trailing */ on a separate line
(style)  do not use C99 // comments